Microsoft.Toolkit.Uwp.UI.Animations/Microsoft.Toolkit.Uwp.UI.Animations.csproj

```diff
@@ -15,22 +15,24 @@
       - AnimationExtensions: Blur, Fade, Light, Offset, Rotate, Saturation, Scale
     </Description>
     <PackageTags>UWP Toolkit Windows Animations Composition Connected Implicit XAML</PackageTags>
+    <LangVersion>9.0</LangVersion>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
   <!-- TODO: Figure out where these go... -->
   <ItemGroup>
     <Compile Remove="Behaviors\**" />
     <Compile Remove="Effects\**" />
```

Comment:
These other things can be removed now, eh?

Reply:
Yup, was just focusing more on completing the migration for all the APIs before focusing on refactoring away the old code and removing all these leftover files, to make sure I didn't miss anything. Will update again when that's done 😊